app/angular/src/server/angular-cli_config.ts

```diff
@@ -47,17 +47,32 @@ function getTsConfigOptions(tsConfigPath: Path) {
 }
 
 export function getAngularCliConfig(dirToSearch: string) {
-  const possibleConfigNames = ['angular.json', 'workspace.json'];
-  const possibleConfigPaths = possibleConfigNames.map((name) => path.join(dirToSearch, name));
-
-  const validIndex = possibleConfigPaths.findIndex((configPath) => fs.existsSync(configPath));
-
-  if (validIndex === -1) {
-    logger.error(`Could not find angular.json using ${possibleConfigPaths[0]}`);
-    return undefined;
+  let angularCliConfig;
+  try {
+    /**
+     * Apologies for the following line
+     * If there's a better way to do it, let's do it
```

Comment:
I don't know if it's better (or if it's possible)🙈 . but an alternative without try/catch would be to let storybook choose if it should use nx or not 🤷‍♂️ 

Something like this: 
`const isNxWorkspace = fs.existsSync(path.join(dirToSearch, 'nx.json'))`

Reply:
The thing is that it will need to access the `readWorkspaceConfig()` from `@nrwl/workspace`, so I apologized for using the `eslint-disable` to let me require it... :)

Reply:
Done

Reply:
okay, 
my point is more about the try/catch. I find that the way we manage the nominal case (without nx) in a Catch is not pretty.
and I propose this solution if it works and suits you of course :) 

```js
const isNxWorkspace = fs.existsSync(path.join(dirToSearch, 'nx.json'))
if(isNxWorkspace) {
 return  require('@nrwl/workspace').readWorkspaceConfig({
      format: 'angularCli',
    });
}
... 

```

Reply:
@ThibaudAV are you ok to go with the current state?

Reply:
Yes 🚀 , I have plans to improve this part. I will do it with a dedicated PR  ;)